### PR TITLE
Added SPM support

### PR DESCRIPTION
**Pull request by Bernix01**

### CODE REVIEW

**Comment by farshadtx** [comment]
Can you please update your PR?

**Comment by Bernix01** [comment]
Done @farshadtx

**Comment by Bernix01** [comment]
Ready to merge 👍

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -22,11 +22,16 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-carthage-
 
+    - name: Swift Package manager
```
Comment (farshadtx) [inline .github/workflows/action.yml:46]: I would suggest adding/duplicate another job here. Renaming `tests` to `carthage-tests` and add `spm-tests` to separate the dependency checks and resolve.

**Review by farshadtx — CHANGES_REQUESTED** [review]
Looks good to me, just some extra changes:
- Please add SPM support section to the README file.
- Update podspec to 2.3.0.
- If you don't mind squash all these commits into one.

**Pull request merged**